src/models/maze.c

```diff
@@ -3,11 +3,14 @@
 char **initialMaze = NULL;
 char **gameMaze = NULL;
 
-SDL_Rect imgMazeSmallCoin = {163, 96, 2, 2};
-SDL_Rect imgMazeBigCoin = {177, 93, 7, 7};
+SDL_Rect imgSmallCoinSprite = {163, 96, 2, 2};
+SDL_Rect imgBigCoinSprite = {177, 93, 7, 7};
 
-SDL_Rect imgMazeOnSprite = {201, 4, 166, 214};
-SDL_Rect imgMazeOnUi = {0, HEADER_SCREEN_HEIGHT, TOTAL_SCREEN_WIDTH, MAZE_SCREEN_HEIGHT};
+SDL_Rect imgMazeBlueSprite = {201, 4, 166, 214};
+SDL_Rect imgMazeBlueUi = {0, HEADER_SCREEN_HEIGHT, TOTAL_SCREEN_WIDTH, MAZE_SCREEN_HEIGHT};
+
+SDL_Rect imgMazeWhiteSprite = {370, 4, 166, 214};
```

Comment:
Instead of specifiying the positions of the sprites 2 times, you can define a X offset to get the white maze

Reply:
https://github.com/Livl-Corporation/livl-pacman/pull/73/commits/759412bb9a86dc2b51690348bf24231b3689ae40